src/sw.ts

```diff
@@ -301,6 +309,10 @@ function isSwConfigGETRequest (event: FetchEvent): boolean {
   return event.request.url.includes('/#/ipfs-sw-config-get')
 }
 
+function isAcceptOriginIsolationWarningRequest (event: FetchEvent): boolean {
+  return event.request.url.includes('/#/ipfs-sw-accept-origin-isolation-warning')
```

Comment:
```suggestion
  return event.request.url.includes('#/ipfs-sw-accept-origin-isolation-warning')
```

or check for `(new URL(request.url)).hash.startsWith('#/ipfs-sw-accept-origin-isolation-warning')`